src/main/java/com/social/accesstoken/services/FbAccessToken.java

```diff
@@ -25,8 +27,14 @@ public FbAccessToken(String url) {
 	public String getAccessToken(SocialNetwork socialNetwork){
 		if(isAuthCorrect(socialNetwork)){
 			log.debug("Login and pass is correct");
-			HtmlPage permissionPage = handleAutorizePage(autorizeEntity, socialNetwork);
-			HtmlPage accessTokenPage = handlePermissionPage(permissionPage);
+			
+			AuthorizationService authorizationService = new AuthorizationService();
+			HtmlPage permissionPage = authorizationService.handleAutorizePage(autorizeEntity, socialNetwork);
+			
+			PermissionService permissionService = new PermissionService(
+					AutorizeDictionary.FORM_ELEMENT_PERMISSION,
+					AutorizeDictionary.FB_BUTTON_NAME);
+			HtmlPage accessTokenPage = permissionService.handlePermissionPage(permissionPage);
```

Comment:
Structure permission page and authorization page is different then we need two services: AuthorizationService and PermissionService